src/app/workspace/[workspaceId]/tasks/page.tsx

```
@@ -22,12 +22,15 @@ const TasksPage = () => {
 	useDocumentTitle("Tasks");
 
 	const workspaceId = useWorkspaceId();
+	if (!workspaceId) return null;
 
 	useTrackActivity({
```

Biome (2.4.15)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)